configuration/changing-shell/en.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Changing Shell"
++++
+# Changing Shell
+
+A Unix shell is a command-line interpreter or shell that provides a traditional user interface for the Unix operating system and for Unix-like systems.
+Users direct the operation of the computer by entering commands as text for a command line interpreter to execute or by creating text scripts of one or more such commands.
```

Comment:
First two paragraphs are blatant copy / paste from Wikipedia. If we could re-word that, that'd be great.